shared-lib/spiutils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+
+name = "spiutils"
+version = "0.1.0"
+authors = [ "Oskar Senft <osk@google.com>" ]
+license = "MIT"
```

Comment:
This is `Apache-2.0` licensed, not `MIT` licensed.

Reply:
Changed. I wasn't sure what this should be set to, I think I copied it from another lib. Thanks for clarifying!